extensions/shared/src/main/java/app/revanced/extension/shared/patches/spoof/SpoofStreamingDataPatch.java

```diff
@@ -23,25 +23,14 @@
 @SuppressWarnings("unused")
 public class SpoofStreamingDataPatch extends BlockRequestPatch {
 
-    /**
-     * key: videoId
-     * value: android StreamingData
-     */
-    private static final Map<String, StreamingDataOuterClass$StreamingData> streamingDataMap = Collections.synchronizedMap(
-            new LinkedHashMap<>(10) {
-                private static final int CACHE_LIMIT = 5;
-
-                @Override
-                protected boolean removeEldestEntry(Entry eldest) {
-                    return size() > CACHE_LIMIT; // Evict the oldest entry if over the cache limit.
-                }
-            });
+    private static final boolean SPOOF_IOS_STREAMING_DATA = SPOOF_STREAMING_DATA
+            && BaseSettings.SPOOF_STREAMING_DATA_TYPE.get() == ClientType.IOS;
```

Comment:
If the default client is not iOS, it will be spoofed as iOS if the following conditions are met:

1. The default client is Android VR and played a Kids video
2. The default client is Android TV and played a Music video or Playlist

It will fall back to the iOS client, but since the default client setting is not iOS, the fix will not be applied in this case

This is why I put the `videoId` and the `clientType` used for spoofing into a HashMap